build-common/NHibernate.props

```diff
@@ -10,6 +10,9 @@
     <AssemblyVersion>$(VersionMajor).$(VersionMinor).0.0</AssemblyVersion>
     <FileVersion>$(VersionPrefix).0</FileVersion>
 
+    <NhAppTargetFrameworks>net461;netcoreapp2.0</NhAppTargetFrameworks>
+    <NhLibTargetFrameworks>net461;netcoreapp2.0;netstandard2.0;</NhLibTargetFrameworks>
```

Comment:
The trailing `;` after netstandard2.0 creates a glitch in Rider (it adds a `.NETFramework, Version=v0.0` target). It should be removed.

Reply:
Shame on Rider for this :)

Should I also add a comment in project files to make it more clear where those NhAppTargetFrameworks/NhLibTargetFrameworks are defined? Something like:
"TargetFrameworks variables are defined in NHibernate.props"

Reply:
No, there is only one include to check.